colcon_cmake/task/cmake/build.py

```diff
@@ -281,17 +281,18 @@ def _get_msbuild_environment(self, args, env):
             env['CL'] = ' '.join(cl_split)
         return env
 
-    def _get_make_arguments(self):
+    def _get_make_arguments(self, env):
         """
         Get the make arguments to limit the number of simultaneously run jobs.
 
         The arguments are chosen based on the `cpu_count`, e.g. -j4 -l4.
 
+        :param env: a dictionary with environment variables
```

Comment:
Please add the parameter type here: `:param dict env: ...`.

Reply:
Nice catch: d3c9c53